payment_icons.gemspec

```diff
@@ -16,6 +16,6 @@ Gem::Specification.new do |s|
   s.test_files = Dir['test/**/*']
 
   s.add_dependency 'frozen_record'
-  s.add_dependency 'rails', '>= 5.0', '< 6.0'
+  s.add_dependency 'rails', '>= 5.0'
```

Comment:
This should be done in a different PR but this gem should not be depending on the entire `rails` framework. It probably only need `railties`.

Reply:
Created https://github.com/activemerchant/payment_icons/issues/171 . I'll take a look.